Formula/youtube-dlc.rb

```diff
@@ -0,0 +1,35 @@
+class YoutubeDlc < Formula
+  desc "Media downloader supporting various sites such as youtube"
+  homepage "https://github.com/blackjack4494/yt-dlc"
+  url "https://github.com/blackjack4494/yt-dlc/archive/2020.11.11-3.tar.gz"
+  sha256 "649f8ba9a6916ca45db0b81fbcec3485e79895cec0f29fd25ec33520ffffca84"
+  license "Unlicense"
+
+  head "https://github.com/blackjack4494/yt-dlc.git"
+
+  livecheck do
+    url "https://github.com/blackjack4494/yt-dlc/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:[.-]\d+)+)["' >]}i)
+  end
+
+  depends_on "make" => :build
```

Comment:
This may not be a necessary dependency but it would be good for someone else to give their thoughts on this.

We have nearly 2,000 formulae (at a minimum) that call `system "make"` but only six include `depends_on "make"`. This formula builds (and tests) fine without this `depends_on` line, so it probably makes sense to remove it.